ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/(dashboard)/page.tsx

```diff
@@ -238,58 +238,57 @@ export default function FunctionDashboardPage({ params }: FunctionDashboardProps
                   )}
                 </div>
               </Block>
-              {function_.configuration?.cancellations &&
-                function_.configuration.cancellations.length > 0 && (
-                  <Block title="Cancellation">
-                    <div className="space-y-3">
-                      {function_.configuration.cancellations.map((cancellation) => {
-                        return (
-                          <NextLink
-                            key={cancellation.event}
-                            href={pathCreator.eventType({
-                              envSlug: params.environmentSlug,
-                              eventName: cancellation.event,
-                            })}
-                            className="border-subtle bg-canvasBase hover:bg-canvasMuted block rounded-md border p-4"
-                          >
-                            <div className="flex min-w-0 items-center">
-                              <div className="min-w-0 flex-1 space-y-1">
-                                <div className="flex min-w-0 items-center">
-                                  <EventsIcon className="text-subtle w-8 shrink-0 pr-2" />
-                                  <p className="truncate font-medium">{cancellation.event}</p>
-                                </div>
-                                <dl className="text-xs">
-                                  {cancellation.condition && (
-                                    <div className="flex gap-1">
-                                      <dt className="text-subtle">If</dt>
-                                      <Tooltip>
-                                        <TooltipTrigger asChild>
-                                          <dd className="truncate font-mono ">
-                                            {cancellation.condition}
-                                          </dd>
-                                        </TooltipTrigger>
-                                        <TooltipContent className="font-mono text-xs">
-                                          {cancellation.condition}
-                                        </TooltipContent>
-                                      </Tooltip>
-                                    </div>
-                                  )}
-                                  {cancellation.timeout && (
-                                    <div className="flex gap-1">
-                                      <dt className="text-subtle">Timeout</dt>
-                                      <dd className="">{cancellation.timeout}</dd>
-                                    </div>
-                                  )}
-                                </dl>
+              {function_.configuration.cancellations.length > 0 && (
```

Comment:
I recommend viewing this diff with whitespace off 
![image](https://github.com/user-attachments/assets/0fdb9d09-2656-46b7-a83f-46c455cb2abf)